config/application.rb

```diff
@@ -4,7 +4,7 @@
 
 if defined?(Bundler)
   # If you precompile assets before deploying to production, use this line
-  Bundler.require(*Rails.groups(:assets => %w(development test)))
+  Bundler.require(*Rails.groups())
```

Comment:
I don't understand what this line did before and why it was changed. The comment talks about production but ten it requires assets from development and test.  Perhaps that was because the assets were precompiled so they should also be required through Bundler. So this change would break that design.

Reply:
From https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-3-2-to-rails-4-0-gemfile

```
9.2 Gemfile
Rails 4.0 removed the assets group from Gemfile. You'd need to remove that line from your Gemfile when upgrading. You should also update your application file (in config/application.rb):

# Require the gems listed in Gemfile, including any gems
# you've limited to :test, :development, or :production.
Bundler.require(*Rails.groups)
```

Reply:
Can you update the comment around this then? And it seems like the commented lines below it should be updated too.